### PR TITLE
Scmo/find by all

### PR DESCRIPTION
**Pull request by scmo**
addressed the issues from the last pull request

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (johnbrush) [inline .classpath]: Test comment...

File: .classpath [hunk 2]
```diff
@@ -1,7 +1,6 @@
...
 <classpath>
 	<classpathentry kind="src" path="src/main/java"/>
-	<classpathentry kind="src" path="src/main/resources"/>
 	<classpathentry kind="src" path="src/test/server/java"/>
```
Comment (johnbrush) [inline .classpath:5]: src/main/resources still missing!

**Comment by scmo** [comment]
i made a space in TestClassA

File: src/test/server/java/ch/zhaw/iwi/cis/kelvin/test/server/TestClassB.java [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
 
+@Entity
 public class TestClassB extends PersistentObject
 {
-	private static final long serialVersionUID = 1L;
-	
-	private String valueOfB;
+	private static final long	serialVersionUID	= 1L;
+
+	private String				valueOfB;
```
Comment (sebastian-stephan) [inline src/test/server/java/ch/zhaw/iwi/cis/kelvin/test/server/TestClassB.java:12]: Are these tabs instead of spaces?

**Pull request merged**